scheduler/pom.xml

```diff
@@ -164,7 +185,8 @@
                 <configuration>
                     <disableXmlReport>true</disableXmlReport>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
-                    <runOrder>random</runOrder>
+<!--                    <runOrder>random</runOrder>-->
```

Comment:
I think the random order was intentional to try to determine race conditions?

Reply:
@KevinJCross not sure if you saw my comment here, but I think this line was deliberately set, presumably so it mimics the ginkgo configuration and runs tests in a random order each time.

Reply:
hi yeah ... I think this change snuck in as I was trying to work out why we fork the jvm for each test class which means spring has to build a new context for each test. This is very slow. 
I will remove this change :)